src/utils/auth.js

```diff
@@ -31,3 +32,28 @@ export const createToken = (payload) =>
  * @returns {Object}
  */
 export const getTokenPayload = (token) => jwt.verify(token, tokenSecret);
+
+/**
+ * Retrieves the userId from the token
+ * @param req
+ * @param authToken
+ * @returns {*}
+ */
+export const getUserId = (req, authToken) => {
```

Comment:
What's the purpose of the authToken second argument here? In what scenario will getUserId be used where this second arg is required

Reply:
That one is for validating an arbitrary token I think. It came with the tutorial lol

Reply:
Can you remove it? If we need it can add it back later.